serializer/custom_normalizer.rst

```diff
@@ -0,0 +1,106 @@
+.. index::
+   single: Serializer; Custom normalizers
+
+How to Create your Custom Normalizer
```

Comment:
```suggestion
How to Create custom Normalizer
```

Reply:
I copied the schema of https://symfony.com/doc/current/serializer/custom_encoders.html ; So are you sure I should be not consistant with existing documentation?

Reply:
Fair enough, lets keep it then 👍